### ISSUE TITLE
Are there any forks or similar packages out there?

### ISSUE THREAD

**Issue by wishinghand** [Wramberg/TerminalView#100]
Since this repo hasn't been updated since 2017, I'm assuming its dead. Has anyone found a similar project that they like to take TerminalView's place?

**Comment by Wramberg** [comment]
@wishinghand Yes it is dead. I spoke with @randy3k a while back about taking over but he would rather start a new project here https://github.com/randy3k/Terminus. I think he is pretty active so I would check that out instead.
